cmd/oci-create-runtime-bundle/main.go

```diff
@@ -114,6 +118,18 @@ func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	//find string complying with regular expression `^[a-zA-Z0-9\-\.\_]+?$`, just for test
+	stringRefFind := refRegexp.FindString(v.ref)
+	v.stderr.Printf("ref name is %s", stringRefFind)
+
+	//check whether string ref match regular expression or not
+	boolRefMatch := refRegexp.MatchString(v.ref)
+	v.stderr.Printf("matching result is %v", boolRefMatch)
+	if boolRefMatch != true {
+		v.stderr.Printf("ref name (%s) is invalid", v.ref)
+		os.Exit(1)
+	}
```

Comment:
There is already [ref-oriented validation in `image/image.go`'s `validate/][1].  Adding that check here saves a few cycles in the `oci-create-runtime-bundle` case, but completely misses the validation check in the `oci-image-validate` case and the “library call to `ValidateLayout`” case.

[1]: https://github.com/TrumanLing/image-tools/blob/adbe39359bf2aac1aa01d3ce376036039c19fe32/image/image.go#L69